corgidrp/data.py

```diff
@@ -1836,8 +1836,9 @@ def unpackbits_64uint(arr, axis):
     Returns:
         _type_: np.ndarray of bits
     """
-    n = np.array(arr).view("u1")
-    return np.unpackbits(n, axis=axis, bitorder='little')
+    arr = arr.astype('>u8')
+    n = arr.view('u1')
+    return np.unpackbits(n, axis=axis, bitorder='big')
 
 def packbits_64uint(arr, axis):
```

Comment:
can we write 1 or 2 unit tests to test the functionality of these functions, even if we aren't using them right now?